MetamodelAlgemeen.md

```diff
@@ -1563,15 +1590,19 @@ Een constraint is gekoppeld aan een klasse waarop ze van toepassing is.
 
 *Toepassing*: objecttype, gegevensgroeptype, relatieklasse.
 
-## Toegestane waarden voor (bepaalde) metadata gegevens
+## Toegestane waarden voor (bepaalde) metagegevens
 
-Tagged values zijn altijd van het datatype CharacterString. Aanvullend geldt:
+### Waardebereik metagegevens
+
+Een aantal metagegevens zijn als tagged values opgenomen. Deze zijn altijd van het datatype CharacterString. Aanvullend geldt:
```

Comment:
niet tagged values, maar als metagegevens (tagged value is een UML uitwerking term)

Reply:
Ik begrijp de opmerking niet. De tekst ging al over tagged values. De vraag is of je die term hier sowieso moet noemen, maar dat werd hier geadresseerd. Ik heb ervan gemaakt dat een aantal metagegevens als tagged value zijn opgenomen. Is aanpassing naar 'een aantal metagegevens zijn in UML als tagged vallue opgenomen' een betere zin

Reply:
Wat ik bedoel is dat we in H2 het niet over tagged values moeten hebben maar over metagegevens en in H3 pas over tagged values. Deze opmerking is niet blokkerend of zo, dus kijk maar wat je ermee wilt doen.

Reply:
verwerkt